### PR TITLE
WIP responsiveness 

### PR DESCRIPTION
**Pull request by adamclark64**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -41,7 +41,9 @@ input#zip{
   font-family: monospace;
   padding: 0px 0px;
   font-size: 11px;
-  margin: 142px 0px 0px 278px;
+  /* left: 39%;
+  top: 36%; */
+  /* margin: 142px 0px 0px 278px; */
```
Comment (gtwalford) [inline css/style.css]: let's just remove these

File: js/input.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ function NewInput(){
         .attr("placeholder","Type Zipcode")
         .attr("maxlength",5)
         .attr("size",10)
-        .style("width","133px")
-        .style("height","18px")
+        // .style("width","10vw")
+        // .style("height","2vw")
```
Comment (gtwalford) [inline js/input.js]: I think we can just delete this

File: js/map.js [hunk 3]
```diff
@@ -162,6 +162,40 @@ function NewMap(){
...
+        .attr("offset", "100%")   
+        .attr("stop-color", "rgb(90, 9, 34)"); 
+
+      viz_g.append("rect")
+        .classed("legend",true)
+        .attr("x",-9)
+        .attr("y",21)
+        .attr("width", 479)
+      	.attr("height", 5)
+      	.style("fill", "url(#linear-gradient)");
```
Comment (gtwalford) [inline js/map.js:198]: fix indentation

**Review by gtwalford — CHANGES_REQUESTED** [review]

File: js/map.js (hunk 3 above)
Comment (adamclark64) [reply js/map.js:198]: idk why this looks like this in the diff but its good

File: css/style.css (hunk 1 above)
Comment (adamclark64) [reply css/style.css]: e26c87e

**Review by gtwalford — APPROVED** [review]

**Pull request merged**